include/fmt/format-inl.h

```diff
@@ -1433,13 +1432,12 @@ extern "C" __declspec(dllimport) int __stdcall WriteConsoleW(  //
     void*, const void*, dword, dword*, void*);
 
 FMT_FUNC bool write_console(std::FILE* f, string_view text) {
-  auto fd = _fileno(f);
+  int fd = _fileno(f);
   if (!_isatty(fd)) return false;
+  std::fflush(f);
```

Comment:
Adding `fflush` here is not correct. This function is called either from the C stream print() or the C++ iostream print(). In the latter case, fflush should not be called at all, instread the ostream flush should be invoked.

Reply:
This flush is indeed redundant for iostream but in the future please report issues like these once to avoid too much notification noise.